### PR TITLE
Add Alpha Support to GetTextures Node

### PR DESCRIPTION
**Pull request by Night1099**
If Alpha is detected in image grab alpha and return as mask

### CODE REVIEW

File: nodes/textures.py [hunk 1]
```diff
@@ -190,6 +202,7 @@ def IS_CHANGED(cls, **kwargs):  # noqa N802
         return float("nan")
 
 
+
```
Comment (nxkb) [inline nodes/textures.py]: just an extra blank line here and above

**Review by nxkb — COMMENTED** [review]
Looks good. Will get this tested on our side... the only think I want to make sure is whether this requires a version bump or whether adding an extra output port is fine for existing graphs.

We fixed our CLA bot, would you mind the CLA for us? (you can just make a comment)

**Comment by Night1099** [comment]
i have tested this in 

an older version:
ComfyUI: 2360 commikt 4ca9b9 (2024-07-12)

and a recent version

ComfyUI: 2641 commit f1c230 (2024-09-01)

With no problems, Shouldnt affect current graphs as its not neccesary to use

How do I do the CLA?

**Comment by nxkb** [comment]
Great. Can you rebase this PR on main and push? I think that will trigger the CLA instructions.

**Comment by Night1099** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by nxkb** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by nxkb — APPROVED** [review]

**Pull request merged**